.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+Fixes #
+
+### Context
+
+
+### Changes Made
+
+
+### Testing?
```

Comment:
I'd drop the question mark so it cannot be read as a binary question and answered "No." 😄